services/grafana-loki/metadata.yaml

```diff
@@ -2,6 +2,8 @@ displayName: Grafana Loki
 description: Horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by Prometheus. Requires Logging Operator.
 category:
   - logging
+dependencies:
+  - rook-ceph-cluster
```

Comment:
for the same reason as here, I think we also need to add the same dependencies block to `velero` metadata.yaml (soft dependency on `rook-ceph-cluster`)

Reply:
Added. cc @gvasquezvargas

Reply:
How do we want to "chain" dependencies? e.g. `rook-ceph-cluster` is defined here but not `rook-ceph` altho both are technically required. But `rook-ceph` is defined as a required dep on `rook-ceph-cluster` -- so does that mean we want to define just the "first" level of deps (so, just `rook-ceph-cluster), and not the entire chain (e.g. `rook-ceph` and `rook-ceph-cluster`)? Not sure which is more confusing to users.

Reply:
☝🏻 just this last remaining question. We have a few places where both `rook-ceph` and `rook-ceph-cluster` are defined and others where just `rook-ceph-cluster` is defined, so just want to be consistent

Reply:
I removed the chaining after discussing it with the UI team. It kind of confuses the process and if we did want to make the user experience better we can still do that programmatically through the UI. Sorry for the back and forth.